spec/factories/permission_templates.rb

```diff
@@ -85,4 +76,36 @@ def self.create_access(permission_template_id, agent_type, access, agent_ids)
       end
     end
   end
+
+  class SourceFinder
+    def self.find_or_create_admin_set(source_id)
+      Hyrax.config.use_valkyrie? ? find_or_create_admin_set_valkyrie(source_id) : find_or_create_admin_set_active_fedora(source_id)
```

Comment:
is `use_valkyrie?` deprecated?

Reply:
I'm still seeing it used in a number of places so I don't think so.  I'm open to using a different approach than `use_valkyrie?` here to determine which model to create.  Do you have any suggestions?

Reply:
oh my mistake, i was remembering seeing https://github.com/samvera/hyrax/blob/main/lib/wings/services/custom_queries/find_file_metadata.rb#L110 but that's specifically for wings queries.